lensit/__init__.py

```diff
@@ -295,15 +295,25 @@ def get_maps_lib(exp, LDres, HDres=14, cache_lenalms=True, cache_maps=False, \
             pix_pha=pixpha, cache_sims=cache_maps)
 
 
-def get_isocov(exp, LD_res, HD_res=14, pyFFTWthreads=4, fn_tensCls=None):
+def get_isocov(exp, LD_res, HD_res=14, pyFFTWthreads=4, do_tensor = False, \
+    fn_scalCls=None, fn_lensCls=None, fn_tensCls=None, r=0.0, lmin=0): 
     """
     Set HD_res to 14 for full sky sampled at res LD.
     """
-    sN_uKamin, sN_uKaminP, Beam_FWHM_amin, ellmin, ellmax = get_config(exp)
-    if fn_tensCls is None:
-        cls_unl, cls_len = get_fidcls(ellmax_sky=ellmax_sky)
+    sN_uKamin, sN_uKaminP, Beam_FWHM_amin, ellmin, ellmax = get_config(exp) 
+    ellmin = lmin 
+
+    if do_tensor:
+        scalCls_unl = get_tenscls(fn_scalCls, ellmax_sky=ellmax_sky)
+        scalCls_len = get_tenscls(fn_lensCls, ellmax_sky=ellmax_sky)
+        tensCls = get_tenscls(fn_tensCls, ellmax_sky=ellmax_sky)
+        cls_unl = scalCls_unl 
+        cls_unl['bb'] += tensCls['bb'] * r / 0.1
+        cls_len = scalCls_len 
+        cls_len['bb'] += tensCls['bb'] * r / 0.1
```

Comment:
Since you are not really reading the r_fid from a file, you might as well just pass the r_fid as an input argument as well, since it is supposed to change depending on the fn_tensCls passed in, unless you want to specify a default fn_tensCls instead. We should avoid hard coding r_fid = 0.1 in the code otherwise.

Reply:
I'm gonna push LensIt/inputs/lensnet_cls/test/r_fixed_1p00e-01/ for future reference.